internal/compiler/generator/rust.rs

```diff
@@ -1044,6 +1044,7 @@ fn generate_sub_component(
             }
 
             pub fn user_init(self_rc: sp::VRcMapped<sp::ItemTreeVTable, Self>) {
+                #![allow(unused)]
```

Comment:
This is a drive by unrelated change i noticed by adding some debug code in some init=> in one of the tests